tests/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+cmake_minimum_required(VERSION 3.14)
+project(TestScheduler)
+
+set(CMAKE_CXX_STANDARD 14)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+
+
+if(NOT DEFINED SCRIPT_BUILD)
+    set(SCRIPT_BUILD OFF CACHE BOOL "Build from script" FORCE)
+endif()
+
+
+add_compile_definitions(TEST_VERSION=1 GTESTING=1)
+
+if(SCRIPT_BUILD)
+    add_compile_definitions(SCRIPT_BUILD=1)
+endif()
+
+add_subdirectory(../external/googletest ./binary_dir)
+include_directories(../external/googletest/include ../src/ ../src/cli/ ../tests)
```

Comment:
`../external/googletest/include` doesn't exist!  Do you mean `../external/googletest/googletest/include`?